src/Resources/scaffolds/6.0/profile/tests/Functional/User/ProfileTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace App\Tests\Functional\User;
+
+use App\Factory\UserFactory;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Zenstruck\Browser\Test\HasBrowser;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Test\ResetDatabase;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class ProfileTest extends KernelTestCase
+{
+    use HasBrowser, Factories, ResetDatabase;
+
+    /**
+     * @test
+     */
+    public function can_update_profile(): void
+    {
+        $user = UserFactory::createOne(['name' => 'Mary Edwards']);
+
+        $this->assertSame('Mary Edwards', $user->getName());
+
+        $this->browser()
+            ->actingAs($user->object())
+            ->visit('/user')
+            ->assertFieldEquals('Name', 'Mary Edwards')
+            ->fillField('Name', 'John Smith')
+            ->click('Save')
+            ->assertOn('/')
+            ->assertSuccessful()
+            ->assertSeeIn('.alert', 'You\'ve successfully updated your profile.')
+        ;
+
+        $this->assertSame('John Smith', $user->getName());
```

Comment:
Maybe just include this first test

Reply:
I kept all the tests but renamed the second one `testValidation` so user's can expand on this.